doc/whats_new/v0.22.rst

```diff
@@ -501,6 +501,14 @@ Changelog
   names by default when a dataframe is passed.
   :pr:`14028` and :pr:`15429` by `Guillaume Lemaitre`_.
 
+- |Fix| Fixed a bug where :func:`inspection.permutation_importance` raised a
```

Comment:
This should be moved above into the log for 0.22.1

Reply:
I have moved it to 0.22.1 change logs